src/Microsoft.CodeAnalysis.Analyzers/UnitTests/MetaAnalyzers/ReleaseTrackingAnalyzerTests.cs

```diff
@@ -754,25 +754,8 @@ class MyAnalyzer : DiagnosticAnalyzer
         // Duplicate changed entries across consecutive shipped releases.
         [InlineData(DefaultShippedHeader + "Id1 | Category1 | Warning |" + BlankLine + DefaultChangedShippedHeader2 + "Id1 | Category1 | Warning | Category1 | Info |" + BlankLine + DefaultChangedShippedHeader3 + "{|RS2006:Id1 | Category1 | Warning | Category1 | Info ||}", "")]
         [Theory]
-        public async Task TestDuplicateEntryBetweenReleasesDiagnostic(string shippedText, string unshippedText)
-        {
-            var source = @"
-using System;
-using System.Collections.Immutable;
-using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.Diagnostics;
-
-[DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
-class MyAnalyzer : DiagnosticAnalyzer
-{
-    private static readonly DiagnosticDescriptor descriptor1 =
-        new DiagnosticDescriptor(""Id1"", ""Title1"", ""Message1"", ""Category1"", DiagnosticSeverity.Warning, isEnabledByDefault: true);
-
-    public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(descriptor1);
-    public override void Initialize(AnalysisContext context) { }
-}";
-            await VerifyCSharpAsync(source, shippedText, unshippedText);
-        }
+        public Task TestDuplicateEntryBetweenReleasesDiagnostic(string shippedText, string unshippedText)
+            => TestDuplicateEntryInReleaseDiagnostic(shippedText, unshippedText);
```

Comment:
I'm not sure if it's a good practice for one test to depend on another test method. I personally prefer a duplication here rather than coupling two tests. Another approach is to make these `InlineData`s and have one test only, but that too will make one test very crowded with many `InlineData`s 😕 

@sharwell for thoughts.

Reply:
We could share the source string as a field/property.

Regarding the `InlineData`s we already have quite some tests with multiple attributes.

Reply:
> I'm not sure if it's a good practice for one test to depend on another test method.

Agreed. Probably better to have a common static helper method that both these tests invoke.